src/tests/cmocka/test_nss_srv.c

```diff
@@ -5308,7 +5309,13 @@ struct netent test_netent = {
     .n_name = discard_const("test_network"),
     .n_aliases = discard_const(test_netent_aliases),
     .n_addrtype = AF_INET,
+#if (__BYTE_ORDER == __LITTLE_ENDIAN)
     .n_net = 0x04030201 /* 1.2.3.4 */
```

Comment:
Perhaps simple `htonl(0x01020304)` instead of `#if...` would be more readable?

Reply:
Similar macros ends up with some function call like `bswap` or similar so it can't really be used in this context (otherwise we get `error: initializer element is not constant`) and I'd rather keep it here then fixing it in places were it is used.

Reply:
Ok.

I restarted CI because it seems previous run failed due to infra issues.